### PR TITLE
fix sig-storage ci config

### PR DESCRIPTION
**Pull request by msau42**
I used https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L316 as a model.

### CODE REVIEW

File: config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml [hunk 1]
```diff
@@ -325,105 +325,71 @@ periodics:
...
     preset-service-account: "true"
     preset-k8s-ssh: "true"
-    preset-bazel-scratch-dir: "true"
-    preset-pull-kubernetes-e2e: "true"
-    preset-pull-kubernetes-e2e-gce: "true"
   spec:
     containers:
     - args:
-      - --root=/go/src
-      - --repo=k8s.io/kubernetes=$(PULL_REFS)
```
Comment (xing-yang) [inline config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml]: We don't need to specify the repo?

Comment (msau42) [reply config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml]: At least in from https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L316, no repo is set.

**Pull request merged**